x-pack/plugin/stack/src/main/java/org/elasticsearch/xpack/stack/StackFeature.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.stack;
+
+import org.elasticsearch.Version;
+import org.elasticsearch.features.FeatureSpecification;
+import org.elasticsearch.features.NodeFeature;
+
+import java.util.Map;
+
+public class StackFeature implements FeatureSpecification {
```

Comment:
Nit: to be consistent with other FeatureSpecification implementation, this should be plural (`StackFeatures`)